src/test/java/org/pesmypetcare/webservice/builders/PathTest.java

```diff
@@ -109,25 +111,29 @@ public void buildForumNamesCollectionPath() {
     @Test
     public void buildPetPath() {
         String path = Path.ofDocument(Collections.pets, userId, petName);
-        assertEquals("users/" + userId + "/pets/" + petName, path, "Should return the path to the pet.");
+        assertEquals("users/" + userId + "/pets/" + petName, path,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/pets/" appears 25 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5482446)